android/build.gradle

```diff
@@ -131,5 +131,5 @@ dependencies {
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.1"
 
     // Appcues
-    implementation "com.appcues:appcues:1.0.0-beta06"
+    implementation "com.appcues:appcues:1.0.0"
```

Comment:
noticed there is no separate version on this android library for the RN wrapper part either, that must be internally managed by RN somehow